open_humans/templates/email/welcome.txt

```diff
@@ -0,0 +1,34 @@
+Thank you for joining the Open Humans community!
+
+To make sure you get future emails from us, add our email to your “contacts”.
+
+{% if newsletter %}You’ve signed up for news and updates, but without this
+step our emails might be “filtered” by your email client.{% else %}
```

Comment:
I considered replacing them with the standard equivalents but they did come through in the email (and look nicer). We can deploy to staging and you can sign up for a new account to get the welcome email and verify that they work for you too?